### PR TITLE
Refine Label Image Rubric type

### PR DESCRIPTION
**Pull request by Myranae**
## Summary:
As part of the Server Side Scoring project, this refactors Label Image's Rubric type based on current usage. It is not currently used in this widget. As we plan to review the use of rubric during LEMS-2440, the Rubric type was left in but changed to `Empty`. It was also added in one place where rubric was listed as a parameter (but was not in use).

Issue: LEMS-2468

## Test plan:
- Confirm all tests pass
- Confirm the label image widget still works in Storybook

### CODE REVIEW

File: packages/perseus/src/validation.types.ts [hunk 1]
```diff
@@ -124,7 +123,9 @@ export type PerseusInteractiveGraphRubric = {
 
 export type PerseusInteractiveGraphUserInput = PerseusGraphType;
 
-export type PerseusLabelImageRubric = PerseusLabelImageWidgetOptions;
+/* TODO(LEMS-2440): Should be removed or refactored. Grading info may need
+    to be moved to the rubric from userInput (mark. */
```
Comment (Myranae) [inline packages/perseus/src/validation.types.ts:127]: I originally just removed everything related to rubric, but I think we might want to pull `answers` out of `userInput`.

File: packages/perseus/src/validation.types.ts [hunk 2]
```diff
@@ -124,7 +123,9 @@ export type PerseusInteractiveGraphRubric = {
 
 export type PerseusInteractiveGraphUserInput = PerseusGraphType;
 
-export type PerseusLabelImageRubric = PerseusLabelImageWidgetOptions;
+/* TODO(LEMS-2440): Should be removed or refactored. Grading info may need
+    to be moved to the rubric from userInput. */
+export type PerseusLabelImageRubric = Empty;
```
Comment (Myranae) [inline packages/perseus/src/validation.types.ts:128]: I originally just removed everything related to rubric, but I think we might want to pull `answers` out of `userInput`.

File: packages/perseus/src/widgets/label-image/label-image.tsx [hunk 3]
```diff
@@ -317,7 +320,8 @@ export class LabelImage
         return {markers};
     }
 
-    showRationalesForCurrentlySelectedChoices(rubric: LabelImageProps) {
+    // TODO(LEMS-2440): Can possibly remove rubric parameter during 2440? Or refactor
```
Comment (handeyeco) [inline packages/perseus/src/widgets/label-image/label-image.tsx:323]: Also kind of related to: https://khanacademy.atlassian.net/browse/LEMS-2519?focusedCommentId=337185

Here's a ticket specifically for this method: https://khanacademy.atlassian.net/browse/LEMS-2544

File: packages/perseus/src/widgets/label-image/label-image-validator.ts [hunk 4]
```diff
@@ -43,14 +43,15 @@ export function scoreMarker(
     return score;
 }
 
+// TODO(LEMS-2440): Can possibly remove rubric parameter during 2440?
```
Comment (handeyeco) [inline packages/perseus/src/widgets/label-image/label-image-validator.ts:46]: For LabelImage I don't think we'll be able to remove rubric. It'll be something like:

```TS
type PerseusLabelImageUserInput = {
    selected: ReadonlyArray<string>;
}

type PerseusLabelImageRubric = {
    answers: ReadonlyArray<string>;
}
```

We just have to untangle how LabelImage is mashing together WidgetOptions/Rubric/UserInput. No action needed right now, it'll be a part of LEMS-2440.

**Review by handeyeco — APPROVED** [review]

File: packages/perseus/src/widgets/label-image/label-image.tsx (hunk 3 above)
Comment (Myranae) [reply packages/perseus/src/widgets/label-image/label-image.tsx:323]: Updated the comment to reference LEMS-2544.

File: packages/perseus/src/widgets/label-image/label-image-validator.ts (hunk 4 above)
Comment (Myranae) [reply packages/perseus/src/widgets/label-image/label-image-validator.ts]: Updated the comment to point out where `answers` is located.

**Pull request merged**